### PR TITLE
Add resource to link a organization to an instance groups

### PR DESCRIPTION
**Pull request by vaxvms**

### CODE REVIEW

File: awx/resource_organization_instance_groups.go [hunk 1]
```diff
@@ -0,0 +1,95 @@
+/*
+*TBD*
+
+# Example Usage
+
+```hcl
+
+	resource "awx_organization_galaxy_credential" "baseconfig" {
```
Comment (denouche) [inline awx/resource_organization_instance_groups.go:8]: awx_organization_galaxy_credential ?

File: awx/resource_organization_instance_groups.go [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+	var diags diag.Diagnostics
+	client := m.(*awx.AWX)
+	awxService := client.OrganizationsService
+	OrganizationID := d.Get("organization_id").(int)
+	_, err := awxService.GetOrganizationsByID(OrganizationID, make(map[string]string))
+	if err != nil {
+		return buildDiagNotFoundFail("organization", OrganizationID, err)
+	}
+
+	result, err := awxService.AssociateInstanceGroups(OrganizationID, map[string]interface{}{
```
Comment (denouche) [inline awx/resource_organization_instance_groups.go:59]: This PR seems to depends on https://github.com/denouche/goawx/pull/32 
Am I right?

File: awx/resource_organization_instance_groups.go (hunk 1 above)
Comment (denouche) [inline awx/resource_organization_instance_groups.go:8]:
```suggestion
	resource "awx_organization_instance_groups" "baseconfig" {
```

File: awx/resource_organization_instance_groups.go (hunk 2 above)
Comment (denouche) [reply awx/resource_organization_instance_groups.go:59]: if so, your PR should depends on denouche/goawx v0.21.0, and not an internal repository from OVH

**Comment by Tioborto** [comment]
Close it in favor of #85

**Pull request closed without merging**